### ISSUE TITLE
Different CMake prefixes for `LCIOTargets.cmake` and `SIOTargets.cmake`

### ISSUE THREAD

**Issue by tmadlener** [iLCSoft/LCIO#183]
When I build latest version v2-20-2, ```LCIOConfig.cmake``` is placed under ```lib/cmake/LCIO/LCIOConfig.cmake``` and ```SIOTargets.cmake``` goes into ```lib/cmake/SIO/SIOTargets.cmake``` (notice the different directories). This leads to the problem when ```LCIOConfig.cmake``` around line 75 looks for ```SIOTargets.cmake``` in the line ```include("${CMAKE_CURRENT_LIST_DIR}/SIOTargets.cmake")```. Of course the easiest way for fix would be to hard-code SIO path as ```include("${CMAKE_CURRENT_LIST_DIR}/.../SIO//SIOTargets.cmake")``` but I'm sure, there is a more elegant way, I'm just not so familiar with CMake.

_Originally posted by @jan-busa in https://github.com/iLCSoft/LCIO/issues/133#issuecomment-1885361438_

**Comment by tmadlener** [comment]
I am fairly certain that this behavior still exists in the current version of the master branch. Would it be possible for you to quickly / easily verify that?

**Comment by jan-busa** [comment]
I have checked out the master and it seems, SIO has been removed from LCIO entirely (moved to separate project). Therefore this problem does not exists for me anymore since ```LCIOConfig.cmake``` now goes via ```find_dependency(SIO REQUIRED)``` and if it does not find it, it builds it as external dependency. I guess, I just need to wait for a new tagged release.

As a side note, the second part of the ```if```, as I have mentioned (```cmake/LCIOConfig.cmake.in``` line 75) still looks for ```SIOConfig.cmake``` in the wrong place but that branch is not currently used (at least for me).
